test/__snapshots__/react.test.js.snap

```diff
@@ -520,15 +537,15 @@ exports[`styled random object expression 1`] = `
 `;
 
 exports[`styled themes 1`] = `
-.glamor-1 {
+.glamor-3 {
   background-color: #ffd43b;
   color: #8c81d8;
   height: 64px;
   font-size: 32px;
 }
 
 <span
-  className="glamor-0 glamor-1"
+  className="glamor-0 glamor-1 glamor-2 glamor-3"
```

Comment:
@mitchellhamilton I'm not sure how to interpret these snapshot diffs. Do they look right to you?